src/hooks/useOrgHooks.js

```diff
@@ -63,19 +61,10 @@ export function useOrganzation() {
   useEffect(() => {
     let cancelled = false
     const fetchOrg = async () => {
-      const organization = await connect(
-        ORG_ADDRESS,
-        [
-          'thegraph',
-          {
-            orgSubgraphUrl: ORG_SUBRAPH_URL,
-          },
-        ],
-        {
-          readProvider: ethereum || ethers,
-          chainId: getDefaultChain(),
-        }
-      )
+      const organization = await connect(ORG_ADDRESS, 'thegraph', {
+        ethereum,
```

Comment:
The name for the provider is now `ethereum`. Also `ethers` should not be passed here, as it is not an Ethereum provider object.